packages/volto/src/components/manage/Widgets/SelectStyling.jsx

```diff
@@ -141,21 +144,21 @@ export const customSelectStyles = {
   }),
   option: (styles, state) => ({
     ...styles,
-    backgroundColor: null,
-    minHeight: '50px',
-    display: 'flex',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    padding: '12px 12px',
+    backgroundColor: state.isFocused ? '#e5f4fb' : null,
     color: state.isSelected
       ? '#007bc1'
       : state.isDisabled
         ? '#b5b5b5'
         : state.isFocused
-          ? '#4a4a4a'
+          ? '#339edc'
           : 'inherit',
+    minHeight: '50px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    padding: '12px 12px',
     ':active': {
-      backgroundColor: null,
+      backgroundColor: state.isFocused ? '#e5f4fb' : null,
```

Comment:
@Abhishek-17h out of curiosity, where did you get the colors in this PR? I know we're picky about being consistent with colors, especially to ensure sufficient contrast for accessibility. I don't know where we capture them all in a single place, like a style guide.

Hopefully someone from the @plone/volto-accessibility team can inform me. If this thing does not exist, it might be a good thing to add to our accessibility PLIP https://github.com/plone/volto/issues/6504.

Reply:
@stevepiercy Thank you for your kind words! I'm glad to contribute to improving Volto. 😊

Reply:
@stevepiercy  Thank you for your kind words! I'm glad to contribute to improving Volto. 😊